Content.Server/GameObjects/EntitySystems/ChemicalReactionSystem.cs

```diff
@@ -0,0 +1,17 @@
+using Content.Shared.Chemistry;
+using Content.Shared.GameObjects.EntitySystems;
+using Robust.Server.GameObjects.EntitySystems;
+using Robust.Shared.Interfaces.GameObjects;
+
+namespace Content.Server.GameObjects.EntitySystems.NewFolder
+{
+    public class ChemicalReactionSystem : SharedChemicalReactionSystem
+    {
+        protected override void OnReaction(ReactionPrototype reaction, IEntity owner, ReagentUnit unitReactions)
+        {
+            base.OnReaction(reaction, owner, unitReactions);
+
+            Get<AudioSystem>().PlayAtCoords("/Audio/Effects/Chemistry/bubbles.ogg", owner.Transform.Coordinates);
```

Comment:
No support for per-reaction effects? Looks like tg has at least like 5 different sounds that are used depending.

Reply:
I didn't realize we had more sound assets available, I will make it pull a file from the prototype.